origami.json

```diff
@@ -40,6 +40,12 @@
 			"template": "demos/src/simple-footer.mustache",
 			"expanded": true,
 			"description": ""
+		},
+		{
+			"name": "pa11y",
+			"template": "demos/src/pa11y.mustache",
+			"expanded": true,
```

Comment:
Should be hidden not expanded. And all the other expanded properties should be removed.